src/templates/cromwell/cromwell-resources.template.yaml

```diff
@@ -276,6 +276,18 @@ Resources:
                 - "logs:PutLogEvents"
                 - "logs:DescribeLogStreams"
 
+        - PolicyName: !Sub CromwellSecretManager-Access-${AWS::Region}
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Resource: "*"
```

Comment:
Can this be scoped down to a specific resource? The the allowed actions, it would be good to keep this to just things Cromwell related.

Reply:
Just did. However ListSecrets still needs the wildcard